src/drivers/libretro/libretro.c

```diff
@@ -713,19 +713,19 @@ void retro_set_environment(retro_environment_t cb)
       { "fceumm_sndvolume", "Sound Volume; 150|160|170|180|190|200|210|220|230|240|250|0|10|20|30|40|50|60|70|80|90|100|110|120|130|140" },
       { "fceumm_sndquality", "Sound Quality; Low|High|Very High" },
       { "fceumm_swapduty", "Swap Duty Cycles; disabled|enabled" },
-      { "fceumm_turbo_enable", "Turbo Enable; None|Player 1|Player 2|Both" },
-      { "fceumm_turbo_delay", "Turbo Delay (in frames); 3|5|10|15|30|60|1|2" },
-      { "fceumm_zapper_mode", "Zapper Mode; lightgun|mouse" },
-      { "fceumm_show_crosshair", "Show Crosshair; enabled|disabled" },
-      { "fceumm_overclocking", "Overclocking; disabled|2x-Postrender|2x-VBlank" },
-      { "fceumm_ramstate", "RAM power up state (Restart); fill $ff|fill $00|random" },
 #ifdef DEBUG
       { "fceumm_apu_1", "Enable Sound Channel 1 (Square 1); enabled|disabled" },
       { "fceumm_apu_2", "Enable Sound Channel 2 (Square 2); enabled|disabled" },
       { "fceumm_apu_3", "Enable Sound Channel 3 (Triangle); enabled|disabled" },
       { "fceumm_apu_4", "Enable Sound Channel 4 (Noise) ; enabled|disabled" },
```

Comment:
You have an empty space at the end of that option title.
It causes all kind of issues like next options not working at all, I did that same mistake once.